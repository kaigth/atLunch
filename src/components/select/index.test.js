import React from "react";
import { render } from '@testing-library/react';
import Select from '.';

describe('Select', () => {
    test('matches snapshot', () => {
        const { container } = render(<Select />)

        expect(container).toMatchSnapshot();
    });
});
