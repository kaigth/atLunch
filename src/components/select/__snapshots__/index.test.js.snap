// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select matches snapshot 1`] = `
<div>
  <div
    class="select"
  >
    <label>
      <input
        name=""
        type="select"
        value=""
      />
    </label>
  </div>
</div>
`;
